src/Downloader.ts

```diff
@@ -278,54 +266,8 @@ function parseHash(hash: string): any {
         }, {});
 }
 
-async function checkForCrashClusters(page: Page): Promise<number> {
-    await sleep(1500).then(() => console.log('Hello after first sleep'));
-    await page.waitFor(4770);
-
-    // TODO waiting for .gwt-Label is not definitive as many pages have these 
-    // CSS selectors, so what's unique yet consistently identifiable AND 
-    // available on this page?
-    await page.waitForSelector('.gwt-Label');
-
-    await sleep(900).then(() => console.log('Hello after second sleep'));
-    await page.waitFor(1004);
-
-    var clusters = 0;
-    const labels = await page.$$eval('.gwt-Label', (el: any[]) => el.map(el => el.textContent));
-    const realTimeCrashesText = labels.filter(textContent => textContent.includes("Real-time crashes"));
-    const screenshotFilename = `realtime-crashes-screenshot_${Date.now()}.png`;
-    if (realTimeCrashesText.length == 0) {
-        console.log("Warning, 'Real-time crashes' message not found, is something wrong? see " + screenshotFilename);
-        await page.screenshot({ path: screenshotFilename, fullPage: true });
-
-    } else {
-        // We're on the right page, hopefully... Now try to get the count. 
-        // The count is only provided when there are 0 crash clusters or less than a page worth.
-        console.log("Info, 'Real-time crashes' message found, see " + screenshotFilename);
-        await page.screenshot({ path: screenshotFilename, fullPage: true });
-
-        const clusterText = labels.filter(textContent => textContent.includes("crash clusters")).toString();
-
-        if (clusterText.length > 0) {
-            clusters = parseInt(clusterText);
-            console.log("Found [" + clusters + "] crash clusters");
-        } else {
-            console.log("No count available, assuming at least a page of crash clusters.")
-            // Could we also save the contents of the page for post-hoc analysis? I've seen 1 crash cluster
-            // shown in  realtime-crashes-screenshot_1572475282166.png yet our code didn't find a match.
-            clusters = -1;  // We assume there are plenty but don't know for sure.
-        }
-    }
-
-    // const publishedPackages = availablePackages.filter(p => p.status == "Published");
-    // const gwtLabels = await page.$$eval('[.gwt-Label]]', (as: any[]) => as.map());
-    // els => els.forEach((el: any) => el.click()));
-    // const crashUrl: string = await row.$eval('.related-link', (el: any) => el.href) as any;
-    return clusters;
-}
-
 async function getCrashClusterIds(page: Page): Promise<string[]> {
-    await page.waitForSelector('[data-type="errorLocation"]'); // loading
+    await page.waitForSelector('[data-type="errorLocation"],section[role=article] > p:not([aria-hidden])'); // loading
 
     const clusterHrefs = await page.$$eval('section table tbody tr a', (as: any[]) => as.map(a => a.href));
```

Comment:
Even when there are no crashes, this will return an empty array, so then none of the following processing is done

Reply:
Sounds good. I had seen issues last week where the code tried to continue to retrieve crash clusters even though there were none (0 crash clusters) which was why I'd added the extra parsing and checking for 0. I love making the code as clear to maintain, operate and debug as we can.